### ISSUE TITLE
Difference between the upstream scheduler main program and the scheduler-plugins one

### ISSUE THREAD

**Issue by yuanchen8911** [kubernetes-sigs/scheduler-plugins#97]
Compared to [upstream kube-scheduler main program](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/scheduler.go), [scheduler-plugin main program](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/cmd/scheduler/main.go) is missing the following code segment, I was wondering why it's removed. Thanks.

```
    // TODO: once we switch everything over to Cobra commands, we can go back to calling
    // utilflag.InitFlags() (by removing its pflag.Parse() call). For now, we have to set the
    // normalize func and add the go flag set by hand.
    pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
    // utilflag.InitFlags()
    logs.InitLogs()
    defer logs.FlushLogs()
```

**Comment by Huang-Wei** [comment]
@yuanchen8911 There is no particular reason. It's ok to add the following lines to scheduler-plugins:

```go
logs.InitLogs()
defer logs.FlushLogs()
```

The `WordSepNormalizeFunc` is not necessary.

**Comment by yuanchen8911** [comment]
> @yuanchen8911 There is no particular reason. It's ok to add the following lines to scheduler-plugins:
> 
> ```go
> logs.InitLogs()
> defer logs.FlushLogs()
> ```
> 
> The `WordSepNormalizeFunc` is not necessary.

Thanks, @Huang-Wei 

Here is `logs.InitLogs` code. Looks like it does the initialization and klog setup. I suspect the klog output won't be written to  `kube-scheduler.log` without the initialization. Anyway, we should probably add it back. I'm opening a PR.   

```
// InitLogs initializes logs the way we want for kubernetes.
func InitLogs() {
    log.SetOutput(KlogWriter{})
    log.SetFlags(0)
    // The default glog flush interval is 5 seconds.
    go wait.Forever(klog.Flush, *logFlushFreq)
}

```

**Comment by yuanchen8911** [comment]
Added `logs.InitLogs()` https://github.com/kubernetes-sigs/scheduler-plugins/pull/100

**Comment by seanmalloy** [comment]
Fixed by #100.

/kind cleanup
/assign @yuanchen8911
/close
